src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,37 +1,117 @@
-import { useState } from 'react';
+import React, { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+interface Props {
+  onAdd: (movie: Movie) => void;
+}
+
+export const NewMovie = ({ onAdd }: Props) => {
```

Comment:
You can use React.FC<Props> instead of writing types each time you use this component
```suggestion
export const NewMovie: React.FC<Props> = ({ onAdd }) => {

//App.tsx
<NewMovie onAdd={movie => handleAddMovie(movie)} />

//instead of 
<NewMovie onAdd={(movie: Movie) => handleAddMovie(movie)} />
```